resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -3,200 +3,432 @@
         "fearlessness",
         "valiance",
         "daring",
-        "bravery"
+        "bravery",
+        "adaptability",
+        "courage",
+        "valorousness",
+        "spirit",
+        "intrigue",
+        "observance"
     ],
     "ambitious": [
         "initiative",
         "loyalty",
         "determination",
-        "bravery"
+        "bravery",
+        "passion",
+        "determination",
+        "commitment",
+        "desire"
```

Comment:
hmmm i'm not sure 'desire' really fits? perhaps 'drive' would work better.  'desire' just doesn't sound quite right.